.ci/jenkins/Jenkinsfile.publish

```diff
@@ -0,0 +1,97 @@
+@Library('jenkins-pipeline-shared-libraries')_
+import org.kie.jenkins.MavenCommand
+
+AGENT_LABEL="kie-rhel7 && kie-mem4"
+MVN_TOOL="kie-maven-3.8.1"
+JDK_TOOL="kie-jdk11"
+REPO="kiegroup-website"
+ORGANIZATION="kiegroup"
+BASE_BRANCH="main"
+MAIL_RECIPIENT="jporter@redhat.com"
+
+pipeline {
+    agent {
+        label "$AGENT_LABEL"
+    }
+    options{
+        timestamps()
+    }
+    tools {
+        maven "$MVN_TOOL"
+        jdk "$JDK_TOOL"
+    }
+    stages {
+        stage('CleanWorkspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('clone kiegroup-website') {
+            steps {
+                script {
+                    checkoutStartkiegroupWeb()
+                }
+            }
+        }
+        stage('Front end compilation') {
+            steps {
+                dir("${REPO}") {
+                    sh 'echo ${NODE_EXTRA_CA_CERTS}'
+                    sh 'npm install --registry=${NPM_REGISTRY_URL} && npm run-script build:webpack'
+                }
+            }
+        }
+        stage('Build website'){
+            steps{
+                dir("${REPO}"){
+                   sh 'mvn clean generate-resources'
+                }
+            }
+        }
+        stage('zip website'){
+            steps{
+                dir("${REPO}"){
+                    sh 'tar -cvzf kiegroup-website.tar -C target/website/ .'
+                    archiveArtifacts '*.tar'
+                }
+            }
+        }
+        stage('publish kiegroup-website to filemgmt.jboss.org') {
+            steps {
+                dir("${REPO}") {
+                    sshagent(["kiegroup-filemgmt"]) {
+                        sh './scripts/rsync_website.sh'
```

Comment:
@mbiarnes can you please clarify the location for this file `./scripts/rsync_website.sh`? Thansk.

Reply:
good point: it is ./build/rsync_website.sh